Upstream: No, should be if applicable
Reason: Fixes location of JAR files

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -216,7 +216,7 @@
     set(LIBRARY_DEST_DIR ${MultiMC_LIBRARY_DEST_DIR})
 
     MESSAGE(STATUS "Compiling for linux system with ${MultiMC_SHARE_DEST_DIR} and MULTIMC_LINUX_DATADIR")
+    SET(MultiMC_APP_BINARY_DEFS "-DMULTIMC_JARS_LOCATION=/usr/${JARS_DEST_DIR}" "-DMULTIMC_LINUX_DATADIR")
-    SET(MultiMC_APP_BINARY_DEFS "-DMULTIMC_JARS_LOCATION=${CMAKE_INSTALL_PREFIX}/${JARS_DEST_DIR}" "-DMULTIMC_LINUX_DATADIR")
 
     # install as bundle with no dependencies included
     set(INSTALL_BUNDLE "nodeps")
